editor.planx.uk/src/@planx/components/Question/Public/Question.tsx

```diff
@@ -24,6 +26,49 @@ export enum QuestionLayout {
 }
 
 const QuestionComponent: React.FC<Question> = (props) => {
+  if (props.forceSelection) {
+    return <VisibleQuestion {...props} />;
+  }
+
+  const [flow, autoAnswerableOptions] = useStore((state) => [
+    state.flow,
+    state.autoAnswerableOptions,
+  ]);
+
+  // Questions without edges act like "sticky notes" in the graph for editors only & can be immediately auto-answered
+  let edges: Edges | undefined;
+  if (props.id) edges = flow[props.id]?.edges
+  if (!edges || edges.length === 0) {
+    return <AutoAnsweredQuestion {...props} answerIds={undefined} />;
+  }
+
+  let idsThatCanBeAutoAnswered: string[] | undefined;
+  if (props.id) idsThatCanBeAutoAnswered = autoAnswerableOptions(props.id);
+
+  if (idsThatCanBeAutoAnswered && idsThatCanBeAutoAnswered.length > 0) {
+    return (
+      <AutoAnsweredQuestion {...props} answerIds={idsThatCanBeAutoAnswered} />
+    );
+  } else {
+    return <VisibleQuestion {...props} />;
+  }
```

Comment:
Very relevant questions & agree there's pieces of code here that already indicate that `Node.tsx` is probably the better place to handle this (eg the initial guard clauses in the store methods you point out), but very in favor of exploring in a follow-up PR to this one because I think it's a larger question than only components I've touched here! 

For example:
- How would it work for Questions, Checklists, Filters updated here?
- How would it work for _other existing component types_ which are always auto-answered / hidden from users like SetValues, Calculates, Pay when fee = 0 etc?

Let's experiment with this _before_ we try out input automations soon :+1: